cafebabel/tests/test_tags.py

```diff
@@ -74,3 +74,24 @@ def test_tag_suggest_wrong_language(client, tag):
     assert response.status_code == HTTPStatus.BAD_REQUEST
     assert (b"Languages available: ['en', 'fr', 'es', 'it', 'de']"
```

Comment:
We could update this to compare with `… in response`

Reply:
I don't get it, what is your suggestion? can you elaborate? (by the way, that's not the PR related to that test)

Reply:
Ah! Never mind, I think I get it, I'll change that upstream.

Reply:
Done in 78dd9e382303cec364fa1658bcb82dbb3343ddad